postprocessing/visualization/tools/extractDataFromUnstructuredOutput.py

```diff
@@ -11,8 +11,9 @@
 parser.add_argument("--add2prefix", help="string to append to prefix for new file", type=str, default="_resampled")
 parser.add_argument("--Data", nargs="+", metavar=("variable"), default=(""), help="Data to resample (example SRs)")
 parser.add_argument("--downsample", help="write one out of n output", type=int)
-parser.add_argument("--float", dest="float", default=False, action="store_true", help="convert to float")
-parser.add_argument("--hdf5", dest="hdf5", default=False, action="store_true", help="output as hdf5")
+parser.add_argument("--double", dest="double", default=False, action="store_true", help="do not convert to float")
+parser.add_argument("--posix", dest="posix", default=False, action="store_true", help="output as posix")
```

Comment:
Maybe make these two arguments a string: like ```"--precision", type=str, choices=["float", "double"],  default="float"```

Reply:
Also the name posix is slightly confusing (has nothing to do with the actual file format). Not sure what (if anything) would be better. Maybe 'raw'?

Reply:
I used posix as it is the name used in xdmfWriterBackend. Do you think posix makes sense in xdmfWriterBackend?
I can use raw (at the end, probably this option will never be used anyway :-p (well I used it as some point when samoa was only reading bin files)).

Reply:
anyway, I've adressed all comments, thx for the review!